packages/lumx-react/src/components/input-helper/InputHelper.stories.tsx

```diff
@@ -0,0 +1,10 @@
+import { InputHelper, Kind } from '@lumx/react';
+import { decorators } from '@lumx/react/story-block';
+import { select, text } from '@storybook/addon-knobs';
+import React from 'react';
+
+export default { title: 'Input Helper', decorators };
+
+export const simpleHelper = ({ theme }) => (
```

Comment:
maybe this is more of a `informationHelper` rather than a `simpleHelper` ? Since the `kind` is `info` ?